app/models/account.rb

```diff
@@ -34,10 +33,6 @@ def source_id_callback(value, _mapping_ids = nil)
     source.nil? ? nil : source.id
   end
 
-  def get_team
-    self.sources.map(&:get_team).flatten.uniq
-  end
```

Comment:
we don't need to get a team for an account anymore?

Reply:
Yes Caio this one depend on calling get_team for Source which depend on ProjectSource at the end